felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -1567,3 +1721,110 @@ func (m *bpfEndpointManager) getInterfaceIP(ifaceName string) (*net.IP, error) {
 	}
 	return nil, errors.New("interface ip address not found")
 }
+
+func (m *bpfEndpointManager) onServiceUpdate(update *proto.ServiceUpdate) {
+	log.WithFields(log.Fields{
+		"Name":      update.Name,
+		"Namespace": update.Namespace,
+	}).Info("Service Update")
+
+	ips := make([]string, 0, 2+len(update.ExternalIps))
+	if update.ClusterIp != "" {
+		ips = append(ips, update.ClusterIp)
+	}
+	if update.LoadbalancerIp != "" {
+		ips = append(ips, update.LoadbalancerIp)
```

Comment:
Wonder if we could get conflicts for LB/Ext IPs in some scenarios.  What if the Ext IP is also added to the node itself, it might hit `lo` :thinking: